experiment/autobumper/main.go

```diff
@@ -42,56 +42,71 @@ const (
 	prowRepo        = "https://github.com/kubernetes/test-infra"
 	testImageRepo   = prowRepo
 	oncallAddress   = "https://storage.googleapis.com/kubernetes-jenkins/oncall.json"
-	githubOrg       = "kubernetes"
-	githubRepo      = "test-infra"
 )
 
-var extraFiles = map[string]bool{
-	"experiment/generate_tests.py": true,
-}
-
-func cdToRootDir() error {
-	if bazelWorkspace := os.Getenv("BUILD_WORKSPACE_DIRECTORY"); bazelWorkspace != "" {
-		if err := os.Chdir(bazelWorkspace); err != nil {
-			return fmt.Errorf("failed to chdir to bazel workspace (%s): %v", bazelWorkspace, err)
+func cdToRootDir(repoDir string) error {
+	d := repoDir
+	if d == "" {
+		if bazelWorkspace := os.Getenv("BUILD_WORKSPACE_DIRECTORY"); bazelWorkspace != "" {
+			if err := os.Chdir(bazelWorkspace); err != nil {
+				return fmt.Errorf("failed to chdir to bazel workspace (%s): %v", bazelWorkspace, err)
+			}
+			return nil
 		}
-		return nil
-	}
-	cmd := exec.Command("git", "rev-parse", "--show-toplevel")
-	output, err := cmd.Output()
-	if err != nil {
-		return err
+		cmd := exec.Command("git", "rev-parse", "--show-toplevel")
+		output, err := cmd.Output()
+		if err != nil {
+			return err
+		}
+		d = strings.TrimSpace(string(output))
 	}
-	d := strings.TrimSpace(string(output))
 	logrus.Infof("Changing working directory to %s...", d)
 	return os.Chdir(d)
 }
 
-type options struct {
-	githubLogin string
-	githubToken string
-	gitName     string
-	gitEmail    string
+type Options struct {
```

Comment:
You can't import from `package main` so you will want to build a subdirectory package to import from.